xbmc/cores/VideoPlayer/DVDInputStreams/DVDFactoryInputStream.cpp

```diff
@@ -40,11 +41,23 @@
 #include "utils/URIUtils.h"
 #include "addons/AddonManager.h"
 #include "addons/InputStream.h"
+#include "Util.h"
 
 
-CDVDInputStream* CDVDFactoryInputStream::CreateInputStream(IVideoPlayer* pPlayer, CFileItem fileitem)
+CDVDInputStream* CDVDFactoryInputStream::CreateInputStream(IVideoPlayer* pPlayer, CFileItem fileitem, bool scanforextaudio)
 {
   std::string file = fileitem.GetPath();
+  if (scanforextaudio)
```

Comment:
No new setting. It's because of the details extraction in DVDFileInfo. Maybe I should remove the bool and also consider ext. audio in DVDFileInfo. What do you think?